From 731e0a3ec424d8b3db1b28047df98887f9582657 Mon Sep 17 00:00:00 2001
From: Han Wang <416810799@qq.com>
Date: Fri, 11 Jan 2019 12:22:16 +0100
Subject: [PATCH 09/11] sdk: Trust: Return TRUST_FEATURE_LEVEL_BAD on
 encryption inactive

 * Prepare to change the logic in Trust frontend.

Change-Id: I8fafdc891bdc17eb92fbfe4be33ed2cd074bad74
---
 .../lineageos/platform/internal/TrustInterfaceService.java   | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/lineage/lib/main/java/org/lineageos/platform/internal/TrustInterfaceService.java b/lineage/lib/main/java/org/lineageos/platform/internal/TrustInterfaceService.java
index 21a55f7..04f6b0a 100644
--- a/lineage/lib/main/java/org/lineageos/platform/internal/TrustInterfaceService.java
+++ b/lineage/lib/main/java/org/lineageos/platform/internal/TrustInterfaceService.java
@@ -322,8 +322,6 @@ public class TrustInterfaceService extends LineageSystemService {
             return TrustInterface.ERROR_UNDEFINED;
         }
 
-        boolean isOldDevice =
-                mContext.getResources().getBoolean(R.bool.config_trustLegacyEncryption);
         int status = policyManager.getStorageEncryptionStatus();
 
         switch (status) {
@@ -333,9 +331,6 @@ public class TrustInterfaceService extends LineageSystemService {
             case DevicePolicyManager.ENCRYPTION_STATUS_ACTIVE_DEFAULT_KEY:
                 return TrustInterface.TRUST_FEATURE_LEVEL_POOR;
             case DevicePolicyManager.ENCRYPTION_STATUS_INACTIVE:
-                return isOldDevice ?
-                        TrustInterface.TRUST_FEATURE_LEVEL_POOR :
-                        TrustInterface.TRUST_FEATURE_LEVEL_BAD;
             case DevicePolicyManager.ENCRYPTION_STATUS_UNSUPPORTED:
                 return TrustInterface.TRUST_FEATURE_LEVEL_BAD;
             default:
-- 
2.17.1

